src/main/java/com/github/markusbernhardt/seleniumlibrary/keywords/Element.java

```diff
@@ -83,15 +84,18 @@ public void currentFrameShouldNotContain(String text, String...params) {
 	@RobotKeyword("Verify the element identified by ``locator`` contains ``text``.\n\r"
 	        + "\n\r"
 	        + "See `Introduction` for details about locators.")
-	@ArgumentNames({ "locator", "text", "message=NONE" })
+	@ArgumentNames({ "locator", "text", "message=NONE", "ignore_case=False" })
 	public void elementShouldContain(String locator, String text, String...params) {
-	    String message = robot.getParamsValue(params, 0, "");
+        String message = robot.getParamsValue(params, 0, StringUtils.EMPTY);
+        Boolean ignoreCase = robot.getParamsValue(params, 1, Boolean.FALSE);
 		String actual = getText(locator);
 
-		if (!actual.toLowerCase().contains(text.toLowerCase())) {
+		if ((!(ignoreCase && StringUtils.containsIgnoreCase(actual, text))) || !(!ignoreCase && StringUtils.contains(actual, text))) {
```

Comment:
Would it be better for readability if we extract all these logical operations into a variable? Something like this:
```java
boolean contains = ignoreCase
    ? StringUtils.containsIgnoreCase(actual, text)
    : StringUtils.contains(actual, text);
```
if we can't use ternary operation and you think that it's not readable, here is the option with simple `if`:
```java
boolean contains = false;
if (ignoreCase) {
    contains = StringUtils.containsIgnoreCase(actual, text);
} else {
    contains = StringUtils.contains(actual, text);
}
```
WDYT?

Reply:
actually, I think the terniary operation should work.
Give me 15 minutes...

Reply:
And pushed...